PowerShell/JumpCloud Module/Private/BulkUser/Wait-JcBulkUserJob.ps1

```diff
@@ -0,0 +1,47 @@
+function Wait-JcBulkUserJob {
+    [CmdletBinding()]
+    param (
+        [Parameter(Position = 0, Mandatory = $true)][ValidateNotNullOrEmpty()][string[]]$jobIds
+    )
+    begin {
+        $finishedJobs = [System.Collections.Generic.List[PSObject]]::new()
+        $unfinishedJobs = [System.Collections.Generic.List[PSObject]]::new()
+    }
+    process {
+        # Initial JobStatus check
+        $JobIds | ForEach-Object {
+            $bulkJobResult = Get-JcSdkBulkUsersResult -JobId $_
+
+            # If job is finished, add to finished list, otherwise add to unfinished list
+            If ($bulkJobResult.Status -eq 'finished') {
```

Comment:
We should add a switch statement here to account for all states:

- finished
- errored
- waiting?

But we can address this later